example-apps/tests-cucumber/exoservice/features/support/world.js

```diff
@@ -0,0 +1,4 @@
+function World() {
+}
+
+module.exports = World;
```

Comment:
I believe we can delete this file and update features/support/env.js to not require the file